src/timeAgo.ts

```diff
@@ -65,8 +64,7 @@ const getSuffix = (type: Unit, value: number, verbose?: boolean): string => {
 			return 'h ago';
 		}
 		case 'd': {
-			if (verbose && shouldPluralise) return ' days ago';
-			if (verbose) return ' day ago';
+			if (verbose) return ' days ago';
```

Comment:
Would this not allow the result "Last updated 1 days ago"?

Reply:
It would return “Last updated yesterday 10:00”. We get hours up to 1 day ago, then “Yesterday HH:MM” up to “two days ago”.

Reply:
Ah ha! Great spot

Reply:
do you fancy adding a comment explaining this? and the same with seconds above (<15 = 'now')?